app/src/main/java/com/example/musicapp/service/InstrumentService.kt

```diff
@@ -11,8 +11,8 @@ import android.util.Log
 
 class InstrumentService {
 
-    fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
-        var _instruments = MutableLiveData<ArrayList<Instrument>>()
+    fun fetchInstruments(): MutableLiveData<ArrayList<Instrument>>{
+        val instruments = MutableLiveData<ArrayList<Instrument>>()
```

Comment:
While we often start private variables with a _, this is declared inside a method, and thus, does not have a visibility modifier.  I'd remove the _.